assets/src/components/useScript/useScript.js

```diff
@@ -0,0 +1,69 @@
+// useScript implementation from: https://usehooks.com/useScript/
+import { useEffect, useState } from 'react';
+
+// Hook
+const loadedScripts = [];
```

Comment:
Suggestion here but we can take it up later too.

We could probably have this check the DOM if the script is already present/loaded to get this info, instead of keeping track of it here. Otherwise if the same script would already be present it would add another script tag with the same source and load it again, potentially causing issues by evaluating it twice.

Reply:
I removed the caching array and now it checks for `!!document.querySelector(`script[src="${src}"]`)`. 👍